### PR TITLE
Edits to README

### PR DESCRIPTION
**Pull request by aoeu**
Adding various grammatical edits and tweaks for consistency in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +85,11 @@ $ go get github.com/glycerine/zygomys/cmd/zygo
 
 [See the wiki for lots of details and a full description of the zygomys language.](https://github.com/glycerine/zygomys/wiki).
 
+### Where did the name zygomys come from?
```
Comment (aoeu) [inline README.md]: This etymology and explanation of the name is very thoughtful and also fun, but I moved it down here so the very first things that a new user sees on the project page is how zygomys can help them solve problems, what zygomys is, and how it works.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
+### What is zygomys?
 
-zygomys allows you to create a Domain Specific Language to drive
-your program with minimal fuss and maximum convenience.
+zygomys is a modernized Lisp with an object-oriented flavor, and
+provides an interpreter and REPL (Read-Eval-Print-Loop;
+that is, it comes with a command line interactive interface).
 
-It is written in Go and plays easily with Go programs
+It is written in Go and plays nicely with Go programs
```
Comment (aoeu) [inline README.md]: I swapped out "easily" for "nicely" riffing off of Rich Hickey's talk "Simple Made Easy" - zygomys inter-operates with Go programs well and in a simple way, but using zygomys may not be easy for someone who's first introduction to Lisp is zygomys.

**Comment by glycerine** [comment]
@aoeu Hey Travis, thanks for these thoughtful suggestions. Some I will adopt, and some not. I'm rather particular about the front page. Still I really appreciate you taking the time make them.

Best,
Jason

**Comment by aoeu** [comment]
Cool! I've been trying to help spot-edit grammar, punctuation, and markdown on wikis and and README's as I read them for the first time and think of anything that might help the authors. I intentionally try to do any documentation changes (for any project, even my own) in really, really tiny commits to accommodate exactly what you ended up doing. I'm glad that any of it was of use!

**Pull request closed without merging**